backend-integration-testing/Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM python:3.11-bullseye
 
+ARG K8S_VERSION=1.29
+
 RUN apt-get -y -qq update && apt-get -qq -y install docker.io libzbar0 apt-transport-https gnupg2 && \
-    curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
-    echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list && \
+    mkdir -p /etc/apt/keyrings && \
+    curl -fsSL https://pkgs.k8s.io/core:/stable:/v${K8S_VERSION}/deb/Release.key | gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg && \
+    echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v${K8S_VERSION}/deb/ /" | tee -a /etc/apt/sources.list.d/kubernetes.list && \
     apt-get update && \
-    apt-get install -y awscli kubectl=1.23.7-00 && \
+    apt-get install -y awscli kubectl && \
```

Comment:
The `awscli` provided in the OS repos, usually is the old v1, while we have to use the awscli v2. 
Maybe considering using this https://github.com/mendersoftware/mender-test-containers/blob/master/aws-k8s-toolbox/Dockerfile as base image or taking the same steps?

Reply:
This PR fixes the build failure related to kubectl. I don't want to carry on other activities/changes as part of this PR. What you suggest makes sense, though is not urgent as fixing the build. If you think it is a good idea, please create a (QA?) ticket and we'll follow up at a later point in time.